docs/self-hosted-appcircle/install-server/kubernetes.md

```diff
@@ -0,0 +1,535 @@
+
+- Add the Appcircle Helm repository.
+
+```bash
+helm repo add appcircle https://helm-package.appcircle.io && \
+helm repo update
+```
+
+- Use the [configured](#create-configuration-file) `values.yaml` file to install the Appcircle Helm chart to your Kubernetes cluster.
+
+:::info
+If you need or want to change the release name, please note that it should be 18 characters or fewer.
+:::
+
+```bash
+helm upgrade --install appcircle-server appcircle/appcircle \
+  --timeout 1200s \
+  -n appcircle --create-namespace \
+  -f values.yaml
+```
+
+The installation process duration depends on factors such as network speed and the processing power of your Kubernetes nodes. Typically, the installation may take up to **10 to 15 minutes**.
+
+If you want to make sure that all containers are **ready to use**, you can use the **`kubectl wait`** command in another terminal window.
+
+```bash
+kubectl wait --for=condition=ready pod \
+  -l app.kubernetes.io/instance=appcircle-server \
+  -n appcircle --timeout 1200s && \
+  echo "Appcircle is ready to use. Happy building! "
+```
+
+When all the pods are **ready**, the command will return with success, and you will see **"Appcircle is ready to use. Happy building!"** message. Now, you are ready to connect to the Appcircle UI and start to build, test and, publish!
+
+## Post-installation Steps
+
+### Create DNS Records
+
+After the Appcircle server installation is finished, you can get the IP addresses of the Appcircle domains and configure the DNS.
+
+You can list the Ingresses with `kubectl` to check the IP address of the Appcircle services domains.
+
+```bash
+kubectl get ingresses -n appcircle
+```
+
+```bash
+NAME                               CLASS   HOSTS                                                          ADDRESS        PORTS      AGE
+appcircle-apigateway               nginx   api.appcircle.spacetech.com,auth.appcircle.spacetech.com       10.45.140.78   80,443     24m
+appcircle-distribution-testerweb   nginx   dist.appcircle.spacetech.com                                   10.45.140.78   80,443     24m
+appcircle-resource                 nginx   resource.appcircle.spacetech.com                               10.45.140.78   80,443     24m
+appcircle-store-web                nginx   *.store.appcircle.spacetech.com                                10.45.140.78   80,443     24m
+appcircle-web-app                  nginx   my.appcircle.spacetech.com                                     10.45.140.78   80,443     24m
+appcircle-web-event                nginx   hook.appcircle.spacetech.com                                   10.45.140.78   80,443     24m
+appcircle-webeventredis            nginx   redis.appcircle.spacetech.com                                  10.45.140.78   80,443     24m
```

Comment:
It will be better to see DNS names as compatible with our DNS section here. (`kvs`)

![Screenshot 2024-12-12 at 11 57 28](https://github.com/user-attachments/assets/5e9217c2-874f-4a65-9dc3-5dc7b3e90273)